lib/cli/run.js

```diff
@@ -241,6 +241,19 @@ exports.builder = yargs =>
       watch: {
         description: 'Watch files in the current working directory for changes',
         group: GROUPS.FILES
+      },
+      'watch-files': {
+        description: 'List of paths or globs to watch',
+        group: GROUPS.FILES,
+        requiresArg: true,
+        coerce: list
+      },
+      'watch-ignore': {
+        description: 'List of paths or globs to exclude from watching',
+        group: GROUPS.FILES,
+        requiresArg: true,
+        coerce: list,
+        default: ['node_modules', '.git']
```

Comment:
Please put this default value in "lib/mocharc.json" and set `default: defaults.watchIgnore`.
Same procedure as `opts` option.

Reply:
https://github.com/mochajs/mocha/pull/3980/commits/f938922b10118ea7f9d58febffb4dec2d54bd0e6